scalahost/nsc/src/main/scala/scala/meta/internal/semantic/ConfigOps.scala

```diff
@@ -2,17 +2,38 @@ package scala.meta.internal
 package semantic
```

Comment:
Shall we split this file into several and then create a dedicated package for all this functionality, like we did with `Vfs.scala` and `Schema.scala`?